src/Sylius/Behat/Context/Setup/PaymentRequestContext.php

```diff
@@ -42,4 +50,37 @@ public function thePaymentRequestActionHasBeenExecutedForOrderWithThePaymentMeth
 
         $this->commandBus->dispatch($addPaymentRequest);
     }
+
+    /**
+     * @Given there is (also) a payment request action :action executed for order :order with the payment method :paymentMethod and state :state
```

Comment:
At first sight, this step and the previous one look similar: `Given the payment request action :action has been executed for order :order with the payment method :paymentMethod`. However, the process of creating the payment request differs in both methods, and IMO this could be confusing for others.

Reply:
changed the step a bit so it differs enough now i think